### DISCUSSION TITLE
Documentation for Zowe VSC extension configuration settings

### DISCUSSION THREAD

**Discussion by mhammack** [zowe/zowe-explorer-vscode#2280] [Q&A] [answered]
If they're out there, I can't find them...at least not the one I want.  I was getting an error for "zowe.settings.version" (which was set to "2.8.0") that it wasn't an integer.  I changed it to 2 and quit getting the error.  However, what is it for? "Track if migration has been processed" doesn't tell me anything and I'm not sure if '2' is correct or if some other value would be better.

To be honest, I don't know how "2.8.0" got in that setting to begin with since I haven't messed with the Zowe settings that much in VSC.

TIA

**Comment by traeok** [comment, marked as answer]
Hi @mhammack,

We use this value to determine whether we need to migrate any old Zowe Explorer settings. However, there is an ongoing issue where Zowe Explorer will set this to the newest version, even though it should be an integer as you mentioned. 

Issue #2166 is currently in progress to resolve this bug, and the fix should be released with the next version of Zowe Explorer 🙂

**Reply by mhammack** [reply]
Thanks!  That's what I needed to know.
